feat: slider (#14)
shivangisharma01/Robust-UI: README.md

## Get Started

> To start using the components in your project, you can add the below `<link>` inside your `<head>` section of your page. Make sure to keep this as your first CSS import, if you would be using your own custom CSS classes.

Reviewer: ```suggestion
To start using the components in your project, you can add the below `<link>` inside the `<head>` section of your page. Make sure to keep this as your first CSS import, to avoid overriding your own css.
```

  <li><a href="https://robust-ui.netlify.app/components/Button/button">Button</a></li>
  <li><a href="https://robust-ui.netlify.app/components/card/card">Card</a></li>
 <li><a href="https://robust-ui.netlify.app/components/Grid/grid">Grid</a></li>
  <li><a href="https://robust-ui.netlify.app/components/ImageR/imageR">Image</a></li>
  <li><a href="https://robust-ui.netlify.app/components/Input/input">Input</a></li>
    <li><a href="https://robust-ui.netlify.app/components/Lists/lists">List</a></li>
  <li><a href="https://robust-ui.netlify.app/components/Navigation/navigation">Navigation bar</a> </li>
    <li><a href="https://robust-ui.netlify.app/components/Rating/rating">Rating</a></li>
      <li><a href="https://robust-ui.netlify.app/components/Slidebar/slidebar">Slide bar</a></li>
  <li><a href="https://robust-ui.netlify.app/components/Toast/toast">Toast</a></li>

</ul>

---
## Overview of each component

### Alert

Alert is a contextual message for typical user actions and notify them of approching results.
<br />
You can check out the various types of alerts [here](https://robust-ui.netlify.app/components/alert/alert)

Types:
* Solid Alert
* Alert with icon

> <hr />

Reviewer: ```suggestion
----
```
